src/v2i-hub/PedestrianPlugin/src/PedestrianPlugin.cpp

```diff
@@ -17,124 +21,145 @@ namespace PedestrianPlugin
  *
  * @param name The name to give the plugin for identification purposes.
  */
-PedestrianPlugin::PedestrianPlugin(string name): PluginClient(name)
+PedestrianPlugin::PedestrianPlugin(const std::string &name) : PluginClient(name)
 {
-	// The log level can be changed from the default here.
-	FILELog::ReportingLevel() = FILELog::FromString("DEBUG");
-
-	AddMessageFilter<MapDataMessage>(this, &PedestrianPlugin::HandleMapDataMessage);
-
-	AddMessageFilter <BsmMessage> (this, &PedestrianPlugin::HandleBasicSafetyMessage);
-
-	// Subscribe to all messages specified by the filters above.
-	SubscribeToMessages();
-}
-
-void PedestrianPlugin::PedestrianRequestHandler(QHttpEngine::Socket *socket)
-{
-	auto router = QSharedPointer<OpenAPI::OAIApiRouter>::create();
-	QString st; 
-	while(socket->bytesAvailable()>0)
-	{	
-		st.append(socket->readAll());
-	}
-	QByteArray array = st.toLocal8Bit();
-
-	char* psmMsgdef = array.data();	
-	// Catch parse exceptions
-    try {
-	    BroadcastPsm(psmMsgdef);
-		writeResponse(QHttpEngine::Socket::Created, socket);
-	}
-	catch(const J2735Exception &e) {
-        PLOG(logERROR) << "Error parsing file: " << e.what() << std::endl;
-		writeResponse(QHttpEngine::Socket::BadRequest, socket);
-	}
+	if (_signSimClient != nullptr)
```

Comment:
Is this UDPClient ever used. If not lets delete it.

Reply:
It's not, removed.